examples/isomorphic/client/webpack.config.js

```diff
@@ -0,0 +1,5 @@
+const mode =
+  process.env.NODE_ENV === 'production' ? 'production' : 'development'
+module.exports = {
```

Comment:
I think it would still help to be explicit about the entry JS file here since not everyone uses webpack and would know that `index.js` automatically gets used

Reply:
Same idea with the output directory. It'll help newcomers if we're explicit I think

Reply:
Makes sense. Can add them to the config file explicitly.
```js
const path = require('path')

const mode =
  process.env.NODE_ENV === 'production' ? 'production' : 'development'
module.exports = {
  mode,
  entry: "./src", // webpack default
  output: {
    path: path.resolve(__dirname, "dist"), // webpack default
    filename: "bundle.js", // webpack default
  }
  
  devServer: {
    proxy: { '/': { target: 'http://localhost:7878' } }
  }
}
```